static/js/browser2/statvar_checkbox.tsx

```diff
@@ -32,16 +36,23 @@ export class StatVarCheckbox extends React.Component<
   StatVarCheckboxPropType,
   StatVarCheckboxStateType
 > {
+  context: ContextType;
   constructor(props: StatVarCheckboxPropType) {
     super(props);
     this.state = {
       checked: false,
     };
-
     this.handleInputChange = this.handleInputChange.bind(this);
   }
 
+  componentDidMount(): void {
+    if (this.props.selected || this.props.statVar in this.context.svPath) {
+      this.setState({ checked: true });
+    }
+  }
+
   private handleInputChange(): void {
+    this.context.togglePath(this.props.statVar, this.props.path);
     this.setState({
       checked: !this.state.checked,
     });
```

Comment:
would be nice for the label to go after the checkbox, so something like
- [ ] Count_Person

Reply:
Done